keras/backend/tensorflow_backend.py

```diff
@@ -169,6 +178,17 @@ def set_session(session):
 # VARIABLE MANIPULATION
 
 def _convert_string_dtype(dtype):
+    """Get the type from a string.
+
+    # Arguments
+        dtype: A string representation of a type.
+
+    # Returns:
+        The type requested.
+
+    # Raises
+        ValueError if `'dtype'` is not supported
```

Comment:
Since `dtype` is an argument name and not a string variable, quotes are not necessary.